.circleci/devcluster/strict-ntsc.devcluster.yaml

```diff
@@ -0,0 +1,57 @@
+stages:
+  - db:
+      name: db
+
+  - master:
+      pre:
+        - sh: make -C tools prep-root
+      config_file:
+        db:
+          host: localhost
+          port: 5432
+          password: postgres
+          user: postgres
+          name: determined
+        checkpoint_storage:
+          type: shared_fs
+          host_path: /tmp
+          storage_path: determined-cp
+        log:
+          level: debug
+        root: tools/build
+        cache: 
+          cache_dir: /tmp/determined-cache
+        security:
+          authz:
+            rbac_ui_enabled: true
+            _strict_ntsc_enabled: true
+            
+  - custom:
+      name: proxy
+      cmd: ["socat", "-d", "-d", "TCP-LISTEN:8081,reuseaddr,fork", "TCP:localhost:8080"]
+      post:
+        - conncheck:
+            port: 8081
+
```

Comment:
if you aren't breaking the connection between agent and master in the test I'd say you can safely omit this proxy step (remove it and use port `8080` in agent configs)